lib/still/preprocessor.ex

```diff
@@ -10,18 +13,60 @@ defmodule Still.Preprocessor do
     OutputPath,
     OutputPath,
     Slime,
-    URLFingerprinting
+    URLFingerprinting,
+    Save,
+    AddLayout,
+    AddContent
   }
 
   @default_preprocessors %{
-    ".slim" => [Frontmatter, Slime, OutputPath],
-    ".slime" => [Frontmatter, Slime, OutputPath],
-    ".eex" => [Frontmatter, EEx, OutputPath],
-    ".css" => [EEx, CSSMinify, OutputPath, URLFingerprinting],
-    ".js" => [EEx, JS, OutputPath, URLFingerprinting],
-    ".md" => [Frontmatter, EEx, Markdown, OutputPath]
+    ".slim" => [AddContent, Frontmatter, Slime, OutputPath, AddLayout, Save],
+    ".slime" => [AddContent, Frontmatter, Slime, OutputPath, AddLayout, Save],
+    ".eex" => [AddContent, Frontmatter, EEx, OutputPath, AddLayout, Save],
+    ".css" => [AddContent, EEx, CSSMinify, OutputPath, URLFingerprinting, AddLayout, Save],
+    ".js" => [AddContent, EEx, JS, OutputPath, URLFingerprinting, AddLayout, Save],
+    ".md" => [AddContent, Frontmatter, EEx, Markdown, OutputPath, AddLayout, Save]
```

Comment:
`AddContent`, `AddLayout` and `Save` are by default all included. They're only here since they can be overriden, but can we abstract it somehow? I don't think we can but I'm just wondering if you have any ideas.

Reply:
Same answer as bellow. I would love to have an abstraction, but I think it's a bit early. This won't last since I'll add image support, RSS, etc, and that will no longer be the case.